src/BloomBrowserUI/lib/long-press/jquery.longpress.js

```diff
@@ -164,13 +170,13 @@
         }
     }
     function showPopup(chars) {
-        popup.empty();
+        popup.find('ul').empty();
```

Comment:
Don't see what this change is for...feels like we should comment and explain wherever we change a lib file